api/v2.0/swagger.yaml

```diff
@@ -1179,7 +1179,7 @@ paths:
         - name: scan_request_type
```

Comment:
Shouldn't we rename the 'scan_request_type'. to 'scanType' to keep it same with stop API?